packages/block-editor/src/components/block-settings/container.native.js

```diff
@@ -30,23 +56,27 @@ function BottomSheetSettings( {
 			contentStyle={ styles.content }
 			{ ...props }
 		>
-			<BottomSheetConsumer>
-				{ ( { currentScreen, extraProps, ...bottomSheetProps } ) => {
-					switch ( currentScreen ) {
-						case colorsUtils.subsheets.color:
-							return (
-								<ColorSettings
-									defaultSettings={ settings }
-									{ ...bottomSheetProps }
-									{ ...extraProps }
-								/>
-							);
-						case colorsUtils.subsheets.settings:
-						default:
-							return <InspectorControls.Slot />;
-					}
-				} }
-			</BottomSheetConsumer>
+			<BottomSheet.NavigationContainer animate>
+				<NavigationContainer theme={ MyTheme }>
+					<Stack.Navigator
+						screenOptions={ {
+							headerShown: false,
+							gestureEnabled: false,
+						} }
+					>
+						<Stack.Screen
+							options={ BottomSheet.NavigationScreen.options }
+							name="Settings"
```

Comment:
Please use subsheet names from [utils](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/mobile/color-settings/utils.native.js#L21) for `Settings` and `Color`

Reply:
I agree that i should use existing consts, however, i feel like we should move them somewhere else because right now I have to import names of block settings Screens from color utils.